app/forms/nsm/steps/disbursement_cost_form.rb

```diff
@@ -19,19 +19,27 @@ def apply_vat
         @apply_vat.nil? ? record.vat_amount.to_f.positive? : @apply_vat == 'true'
       end
 
+      # rubocop:disable Metrics/MethodLength
       def calculation_rows
-        [
-          [translate(:before_vat), translate(:after_vat)],
-          [{
-            text: NumberTo.pounds(total_cost_pre_vat),
-            html_attributes: { id: 'total-without-vat' }
-          },
-           {
-             text: NumberTo.pounds(total_cost),
-             html_attributes: { id: 'total-with-vat' },
-           }],
-        ]
+        # this logic is needed to prevent invalid values erroring in the NumberTo methods
+        if total_cost_pre_vat.is_a?(String)
+          record.errors.add(:total_cost_without_vat, :not_a_decimal)
+          []
```

Comment:
I'm also not sure we want this method to return an empty array, as that will result in the UI doing something new, i.e. showing a table with no body. We probably want this to do what it would do if the `total_cost_without_vat` string was left blank, i.e. return the same structure just with empty `text` attributes.